### DISCUSSION TITLE
DS1621+ on DSM 7.2.1-69057 Update 5 with 2x UGREEN USB C 2.5G LAN Adapter (B0CD1FDKT1) and MBP 13" (2020) works fine!!

### DISCUSSION THREAD

**Discussion by sfuertbauer** [bb-qq/r8152#394] [Show and tell]
Installed the driver as documented, started "RTL8152/8153 Driver" in Package Manager (-> missing step!) and LAN #5 appeared quite instantly. Configured manual IPs (192.168.7.0/24) with Gateway accordingly (opposite IPs) and recommended MTU, installed iperf3 on both NAS and Mac and got 2.27Gbit/s.

**Comment by TheS1R** [comment]
Use a USB-A to USB-C adapter.
